src/WattleScript.Interpreter/Tree/Statements/ClassDefinitionStatement.cs

```diff
@@ -200,8 +242,14 @@ public override void ResolveScope(ScriptLoadingContext lcontext)
             tostringClosure.ResolveScope(lcontext);
             //functions
             foreach(var fn in functions)
-                fn.exp.ResolveScope(lcontext);
+                fn.Value.Expr.ResolveScope(lcontext);
```

Comment:
Resolve instance functions here, then resolve static functions in the same place we resolve static fields

Reply:
resolved in ab2872c